.github/dependabot.yml

```diff
@@ -6,7 +6,7 @@ updates:
     interval: weekly
     timezone: America/Los_Angeles
     day: tuesday
-  target-branch: "develop"
+  target-branch: "default"
```

Comment:
I'm not sold that "default" is _better_ than `develop` as a branch name. Agree about `master` -> `release`. But I suppose it might make the GitHub "default" branch appear more "stable" if it's called `default`.

Reply:
That's my thought, I want it abundantly clear that this is the branch to merge in and out of, for all the people not reading CONTRIBUTING.md 😁

Reply:
We should also add a PR template 😁